### PR TITLE
Add Typelevel Summit 2023 event

### PR DESCRIPTION
**Pull request by valencik**
Adds an events page for the Typelevel Summit 2023

### CODE REVIEW

File: collections/_events/2023-10-26-summit-nescala.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+layout: event
+
+title: "Typelevel Summit NEScala"
+short_title: "Typelevel Summit"
+date_string: "October 26, 2023"
+location: "Online"
+description: "One day of recorded talks and one day of unconference, co-located with NE Scala."
```
Comment (jducoeur) [inline collections/_events/2023-10-26-summit-nescala.md:8]:
```suggestion
description: "One day of talks and one day of unconference, co-located with NE Scala."
```
Calling it "recorded talks" makes it sounds like the talks will all be pre-recorded, which I don't think is true -- while we're going to allow for pre-recorded, I expect most to be live, and for some to have Q&A.  Tweaked the section below to clarify (I think).

File: collections/_events/2023-10-26-summit-nescala.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+## About the Summit
+
+The tenth Typelevel Summit will once again be co-hosted with the [Northeast Scala Symposium](http://www.nescala.org/) virtually, with one day of recorded talks and one day of (shared) unconference.
+
+The schedule for this year is as follows:
+
+* October 26 (Thursday): Typelevel Summit
+* October 27 (Friday): NE Scala 2020
```
Comment (jducoeur) [inline collections/_events/2023-10-26-summit-nescala.md:19]:
```suggestion
* October 27 (Friday): NE Scala 2023
```
Do you know where you copied this from?  Sounds like we might need to fix something somewhere.

File: collections/_events/2023-10-26-summit-nescala.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+short_title: "Typelevel Summit"
+date_string: "October 26, 2023"
+location: "Online"
+description: "One day of recorded talks and one day of unconference, co-located with NE Scala."
+
+---
+
+## About the Summit
+
+The tenth Typelevel Summit will once again be co-hosted with the [Northeast Scala Symposium](http://www.nescala.org/) virtually, with one day of recorded talks and one day of (shared) unconference.
```
Comment (jducoeur) [inline collections/_events/2023-10-26-summit-nescala.md:14]:
```suggestion
The tenth Typelevel Summit will once again be co-hosted with the [Northeast Scala Symposium](http://www.nescala.org/) virtually, with one day of talks and socializing online and one day of unconference.  Talks will be recorded and available online afterwards.
```

**Review by jducoeur — COMMENTED** [review]
A couple of tweaks, but otherwise looks good -- thanks!

File: collections/_events/2023-10-26-summit-nescala.md (hunk 3 above)
Comment (SethTisue) [reply collections/_events/2023-10-26-summit-nescala.md:14]: maybe "discussion and socializing"?

Comment (SethTisue) [reply collections/_events/2023-10-26-summit-nescala.md:14]: or "one of day of talks and one day of unconference, with discussion and socializing online throughout"? idk

Comment (jducoeur) [reply collections/_events/2023-10-26-summit-nescala.md:14]: I like that second version!

**Comment by valencik** [comment]
Updated based on feedback
Marking ready for review. I have not built this locally, as I don't have a working bundler setup.

**Review by jducoeur — APPROVED** [review]

**Pull request merged**